Added docs on WAF (#3587)
umbraco/UmbracoDocs: Umbraco-Cloud/Set-Up/Manage-Hostnames/index.md

On the Professional and Enterprise plan, you can manually add your own certificate to your Umbraco Cloud project and bind it to one of the hostnames you've added.

> ## [Using WAF on Cloud](WAF)

Reviewer: Maybe the heading should be "Using your own WAF on Umbraco Cloud" ?

> ## [Using WAF on Cloud](WAF)

Author: ```suggestion
## [Using your own WAF on Umbraco Cloud](WAF)
```

> There is an option to install Web Application Firewall if that is needed.

Reviewer: Change this to something like:
"If you need to use your own Web Application Firewall (WAF) in front of your Umbraco Cloud website then this section will highlight some of the most common configuration needed. But please note that configuration may vary depending on which WAF you are using, so you should always consult your vendor for best practices and recommendations."

> There is an option to install Web Application Firewall if that is needed.

Author: If you need to use your own Web Application Firewall (WAF) in front of your Umbraco Cloud website then this section will highlight some of the most common configuration needed. But please note that configuration may vary depending on which WAF you are using, so you should always consult your vendor for best practices and recommendations.

> There is an option to install Web Application Firewall if that is needed.

Author: ```suggestion
If you need to use your own Web Application Firewall (WAF) in front of your Umbraco Cloud website then this section will highlight some of the most common configuration needed. But please note that configuration may vary depending on which WAF you are using, so you should always consult your vendor for best practices and recommendations.
```

> If you are using a custom certificate for that specific hostname with WAF make sure you are able to upload the custom certificate:

Reviewer: Change to "In most cases you need to ensure that the WAF and Umbraco Cloud are using the same certificate on the specific hostname. Custom certificates is a plan-specific feature on Umbraco Cloud, so please make sure that you have access to upload certificates."

> 1. Make sure the hostname is pointing to  Umbraco Cloud (dns. umbraco.io)

Reviewer: There are too many spaces in this line.....change to:
"Make sure the hostname is pointing to Umbraco Cloud (dns.umbraco.io)"

> 1. Make sure the hostname is pointing to  Umbraco Cloud (dns. umbraco.io)

Author: ```suggestion
Make sure the hostname is pointing to Umbraco Cloud (dns.umbraco.io)
```

2. Certificates are issued for the actual hostname - It is probably required to have a custom certificate for a WAF hostname. 
3. Be on a plan that supports custom certificates

> Even though the hostname isn't pointing to Umbraco Cloud (dns.umbraco.io) can the certificate be validated by adding a txt record for the hostname. Or a DNS record.

Reviewer: Suggestion: "When configuring the hostname and certificate on Umbraco Cloud it will be necessary to validate the hostname using a TXT record. This is needed, because in most cases the WAF will hide that the website is running on Umbraco Cloud, which means that the usual domain ownership verification cannot be performed. This same approach can also be used to configure a hostname prior to updating the DNS for the hostname."

> Even though the hostname isn't pointing to Umbraco Cloud (dns.umbraco.io) can the certificate be validated by adding a txt record for the hostname. Or a DNS record.

Author: ```suggestion
"When configuring the hostname and certificate on Umbraco Cloud it will be necessary to validate the hostname using a TXT record. This is needed, because in most cases the WAF will hide that the website is running on Umbraco Cloud, which means that the usual domain ownership verification cannot be performed. This same approach can also be used to configure a hostname prior to updating the DNS for the hostname.
```

When that is validated, then the hostname will, in principle, be ready to be used in Umbraco Cloud.

> So when you update your DNS, regarding go-live, will it typically be very fast and the hostname will be ready to use.

Reviewer: Don't think this is necessary. It becomes a bit confusing as its mixing two different things.

> When that is validated, then the hostname will, in principle, be ready to be used in Umbraco Cloud.

Author: ```suggestion

```

> So when you update your DNS, regarding go-live, will it typically be very fast and the hostname will be ready to use.

Author: ```suggestion

```
